### ISSUE TITLE
Dependencies not installed in wpdtrt test

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-npm-scripts#20]
When run from wpdtrt-npm-scripts mocha tests, the `wpdtrt/vendor` and `wpdtrt/node_modules` folders are not generated.

This theme is a dependency of wpdtrt-dbth, so it's possible that it is somehow bypassing the postinstall script.

```
1) scripts
       wordpress-parent-theme
         test
           runs without error:

      AssertionError: expected './scripts/test/phpunit.sh: line 22: ./vendor/bin/phpunit: No such file or directory\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! wpdtrt-npm-scripts@0.2.1 test: `./scripts/test/phpunit.sh`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the wpdtrt-npm-scripts@0.2.1 test script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\nnpm WARN Local package.json exists, but node_modules missing, did you mean to install?\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/dan/.npm/_logs/2020-04-23T04_28_40_241Z-debug.log\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 1\nnpm ERR! wpdtrt@0.2.15 test: `npm run test --prefix $npm_package_config_wpdtrt_npm_scripts`\nnpm ERR! Exit status 1\nnpm ERR! \nnpm ERR! Failed at the wpdtrt@0.2.15 test script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /Users/dan/.npm/_logs/2020-04-23T04_28_40_262Z-debug.log' to equal ''
      + expected - actual

      -./scripts/test/phpunit.sh: line 22: ./vendor/bin/phpunit: No such file or directory
      -npm ERR! code ELIFECYCLE
      -npm ERR! errno 1
      -npm ERR! wpdtrt-npm-scripts@0.2.1 test: `./scripts/test/phpunit.sh`
      -npm ERR! Exit status 1
      -npm ERR! 
      -npm ERR! Failed at the wpdtrt-npm-scripts@0.2.1 test script.
      -npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
      -npm WARN Local package.json exists, but node_modules missing, did you mean to install?
      -
      -npm ERR! A complete log of this run can be found in:
      -npm ERR!     /Users/dan/.npm/_logs/2020-04-23T04_28_40_241Z-debug.log
      -npm ERR! code ELIFECYCLE
      -npm ERR! errno 1
      -npm ERR! wpdtrt@0.2.15 test: `npm run test --prefix $npm_package_config_wpdtrt_npm_scripts`
      -npm ERR! Exit status 1
      -npm ERR! 
      -npm ERR! Failed at the wpdtrt@0.2.15 test script.
      -npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
      -
      -npm ERR! A complete log of this run can be found in:
      -npm ERR!     /Users/dan/.npm/_logs/2020-04-23T04_28_40_262Z-debug.log
      
      at Context.<anonymous> (/Users/dan/Websites/wpdtrt-npm-scripts/test/tasks/scripts.spec.js:471:55)



npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wpdtrt-npm-scripts@0.2.1 test:internal: `chmod a+x test/scripts/*.sh && ./test/scripts/teardown.sh && ./test/scripts/setup.sh && mocha --recursive "./test/**/*.spec.js" --full-trace`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wpdtrt-npm-scripts@0.2.1 test:internal script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dan/.npm/_logs/2020-04-23T04_28_40_315Z-debug.log
```

**Comment by dotherightthing** [comment]
When run from wpdtrt directly, `npm install` does generate `node_modules` and `vendor`.

`npm run test` hangs first time, but completes the second time.

**Comment by dotherightthing** [comment]
Fixed with #22
